LidarScan:
  ros__parameters:


ObstacleAvoidance:
  ros__parameters:
    min_obstacle_distance: 0.5
    max_obstacle_distance: 0.7
    sos_distance: 0.3

IMUDriverNode:
  ros__parameters:
    serial_port: "/dev/imu_usb"
    baudrate: 9600
    imu_topic: "imu/og_data_raw"
    mag_topic: "imu/og_mag"